### DISCUSSION TITLE
Plan to migrate ELAN plugin?

### DISCUSSION THREAD

**Discussion by ArthLeu** [katspaugh/wavesurfer.js#3072] [Ideas]
Do we have any plans to migrate [the ELAN plugin](https://wavesurfer-js.org/example/elan-wave-segment/) to the new WS v7, or is it already compatible?

I have a project using ELAN and WS v6, but I want to upgrade to WS v7 because it's so much better. Wondering if this is possible.

**Comment by katspaugh** [comment]
Sure, we could do that.
I think the part of the plugin that renders a table is better handled on the application level, but I can see value in parsing the ELAN XML into JSON and rendering annotations as regions.

**Comment by ArthLeu** [comment]
Looking forward to that addition.

**Comment by bhogan-mitre** [comment]
Hello, is 6.6.4 the latest version that includes the ELAN plugin?  I didn't see mention of ELAN getting dropped in the [7.0.0 notes](https://github.com/katspaugh/wavesurfer.js/releases/tag/7.0.0), but that's what is sounds like from this thread.  

I use the ELAN plugin heavily and this would prevent me from updating any further than that.  Thanks.
